spec/classes/init_spec.rb

```diff
@@ -104,7 +103,7 @@
           context 'when httpd is set to valid bool <true>' do
             let(:params) { { httpd: true } }
 
-            content = <<-END.gsub(%r{^\s+\|}, '')
+            content = <<~END
               |set httpd port 2812 and
```

Comment:
The `|` part can be dropped. It was done as part of the "heredoc" implementation (see gsub).